src/core/skills-detect.ts

```
@@ -1,39 +1,644 @@
   try {
-    await access(skillsPath);
-    return ['skills/'];
+    await access(targetPath);
+    return true;
+  } catch {
+    return false;
+  }
+}
+
+async function collectSkillKeys(rootPath: string): Promise<string[]> {
+  let dirents;
```

Biome (2.4.11)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)